src/components/EditItemForm.tsx

```
@@ -44,16 +45,31 @@ export function EditItemForm({ item, onClose, onSuccess }: EditItemFormProps) {
     setError(null);
 
     try {
+      // Validate stock is a number
+      const stockNum = parseInt(stock);
+      if (isNaN(stockNum) || stockNum < 0) {
+        throw new Error('El stock debe ser un número válido mayor o igual a 0');
+      }
+
+      // Validate price is a number
+      const priceNum = parseFloat(price);
+      if (isNaN(priceNum) || priceNum < 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)